samples/SentimentAnalyzerCustomSkill/cpp/Apps/FaceSentimentAnalysisApp_C++WinRT/FaceSentimentAnalysisApp_C++WinRT.vcxproj

```diff
@@ -31,7 +31,7 @@
     <ProjectGuid>{425CE54A-C887-45C5-94A7-72738F49FE11}</ProjectGuid>
     <Keyword>Win32Proj</Keyword>
     <RootNamespace>FaceSentimentAnalysiscpp</RootNamespace>
-    <WindowsTargetPlatformVersion>10.0.18898.0</WindowsTargetPlatformVersion>
+    <WindowsTargetPlatformVersion>10.0.18925.0</WindowsTargetPlatformVersion>

```

Comment:
>18925 [](start = 39, length = 5)

Is this public preview version?

Reply:
This is the internal build we selected to perform sample code updates. The changes are about a week away from going to WIP.